include/drivers/adp536x.h

```diff
@@ -19,3 +19,4 @@ int adp536x_buckbst_enable(bool enable);
 int adp536x_buck_1v8_set(void);
 int adp536x_buckbst_3v3_set(void);
 int adp536x_factory_reset(void);
+int adp536x_oc_chg_current_set(u8_t value);
```

Comment:
Really do not like the fact that this file is totally undocumented. But it is probably a note to the author. I have notified @ru-fu and @b-gent about this fact.